index.html

```diff
@@ -0,0 +1,140 @@
+				<a href="#about">About Me</a>
+				<a href="#portfolio">My Work</a>
+				<a href="#contact">Contact</a>
+			</div>
+		</div>
+	</nav>
+
+
+	<!-- yellow introductory banner -->
+	<header>
+		<div class="container_main">
+			<div class="callout">
+				<h2> hey there! <br />
+					<span class="callout_right">nice to see you...</span></h2>
+			</div>
+		</div>
+	</header>
+
+
+	<!-- about me section -->
+	<div id="about" class="container_main">
+		<div class="about_me_box">
+			<div class="about_me">
+				<h3>About me</h3>
+				<p> Hi, I&#39;m Hannah! After many years working as a Graphic Designer,
+					I decided to make a career change (an idea both exciting and slightly terrifying!).
+					I&#39;m currently training to be a full stack developer at Mayden Academy in Bath, a 16 week
+					intensive course to learn the skills required for the industry.
+				</p>
+				<p>Please take a look at my work completed so far... </p>
+			</div>
+			<div class="img_me_box">
+				<div class="img_me"></div>
+			</div>
+		</div>
+	</div>
+
+	<!-- portfolio section -->
+	<div id="portfolio" class="container_main">
+		<h3>My work</h3>
+		<div class="portfolio_box">
+			<div class="box1">
+				<div class="overlay">
+					<div class="overlay_contents">
+						<div class="text">Mayden Academy Logo</div>
+						<a href="./portfolio.html#mayden_logo" class="portfolio_link">Take a look</a>
+					</div>
+				</div>
+			</div>
+			<div class="box2">
+				<div class="overlay">
+					<div class="overlay_contents">
+						<div class="text">Pilot Shop Webpage</div>
+						<a href="./portfolio.html#pilot_shop" class="portfolio_link">Take a look</a>
+					</div>
+				</div>
+			</div>
+			<div class="box3">
+				<div class="overlay">
+					<div class="overlay_contents">
+						<div class="text">TBC<br /> Watch this space... </div>
+					</div>
+				</div>
+			</div>
+			<div class="box4">
+				<div class="overlay">
+					<div class="overlay_contents">
+						<div class="text">TBC<br /> Watch this space... </div>
+					</div>
+				</div>
+			</div>
+			<div class="box5">
+				<div class="overlay">
+					<div class="overlay_contents">
+						<div class="text">TBC<br /> Watch this space... </div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+
+
+	<!-- contact section -->
+	<div id="contact"></div>
+	<div class="contact_panel">
+		<div class="container_main">
+			<div class="center_text">
+				<h3>Get in touch</h3>
+				<p>I'd love to hear from you!</p>
+				<div class="contact_panel_a">
+					<a class="email" href="mailto:hannah.golightly01@gmail.com?subject=hello">hannah.golightly01@gmail.com</a>
+				</div>
+				<div class="social_media_box">
+					<a href="https://github.com/Hannahrey">
+						<img src="./images/GitHub-Mark-120px-plus.png" alt="GitHub-Logo"/>
+					</a>
+				</div>
+				<div class="social_media_box">
+					<a href="https://www.linkedin.com/in/hannah-g-2b053452/">
+						<img src="./images/linkedin.png" alt="Linkedin-Logo"/>
+					</a>
```

Comment:
It might be worth making these social media links open in a new tab to stop users from being taken away from your site. You can do that by adding a `target="_blank" attribute to the a tags.

Reply:
thanks - added in!